src/main/java/seedu/exercise/ui/BarChartPanel.java

```diff
@@ -44,18 +44,19 @@ private void display() {
         barChart.setAnimated(false);
         barChart.layout();
 
-        xAxis.setLabel(statistic.getCategory());
+        xAxis.setLabel(ChartTextUtil.firstLetterUpperCaseFormatter("exercises"));
```

Comment:
consider changing the name of this method. Sounds like you get a formatter back. Maybe consider `ChartTextUtil.changeFirstLetterToUpperCase`

Reply:
ok